examples/chef/nrfconnect/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ include(${CHEF}/project_include.cmake)
 
 get_filename_component(GEN_DIR ${CHEF}/out/${SAMPLE_NAME}/zap-generated REALPATH)
 
-set(CONF_FILE prj.conf)
+set(CONF_FILE ${CHIP_ROOT}/config/nrfconnect/app/sample-defaults.conf prj.conf)
```

Comment:
`sample-defaults.conf` not longer exists. Does it even compile?

Reply:
Please note this is for a special interop branch and not master. All these changes were done to make it compile on the branch.

Reply:
Ok, sorry, I didn't notice that.